### ISSUE TITLE
[Feature Request]: Shortcut for Profile Selection

### ISSUE THREAD

**Issue by jason-storey** [FreeTubeApp/FreeTube#5839]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

I have about 20 categories I view youtube under. from education to entertainment split by long form, short form as well as music and tutorials, video essays, etc

I use my keyboard and not my mouse 90% of the time. vim, flowlauncher, autohotkey etc. FreeTube is one of 3 apps that have heavy mouse use requirements for me. 

### Proposed Solution

I would love to be able to step through the profiles while on the subscriptions page with ctrl+left and right and/or ctrl+1-9 to quick jump to the profile at that index. 

Basically any way to support the workflow of:

"I have some time spare, its roughly [duration] I can catch up on [topic]" and press the appropriate key binding to get to that collection of new relevant content.

### Alternatives Considered

I had a look to see if there was a rest api, Resource URIs I could link to with espanso or other means of driving it myself. 
Aside from writing lengthy autohotkey scripts based on captures of the ui I unfortunately don't see a way I could cludge it together myself. 

### Issue Labels

ease of use improvement, improvement to existing feature, new keyboard shortcut

### Additional Information

Thanks again, Amazing app I've only been using it a month or so properly and I could really see myself replacing all youtube use with it.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
closing in favor of https://github.com/FreeTubeApp/FreeTube/issues/2691 and https://github.com/FreeTubeApp/FreeTube/issues/1864
